spec/requests/posts_spec.rb

```diff
@@ -1,7 +1,54 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
+  let!(:user) do
+    User.create(
+      name: 'Rails',
+      photo: 'http://localhost:3000/thursday.png',
+      bio: 'Something test',
+      postsCounters: 10
+    )
+  end
+
+  let!(:post) do
+    Post.create(
+      title: 'Ruby',
+      text: 'Learn Ruby',
+      author: user,
+      comments_counter: 10,
+      likes_counter: 10
+    )
+  end
+
   describe 'GET /index' do
-    pending "add some examples (or delete) #{__FILE__}"
+    before do
+      get "/users/#{user.id}/posts/"
+    end
+
```

Comment:
- [ ] Kindly make sure to have a test for the `post#index` action that checks response status was correct as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20response%20status%20was%20correct.).

Reply:
@gemmen29 please i need more clarification, Because i have done the requirement

Reply:
Hi @gemmen29  @katfogy added  the specs for post#index action already, this will be approved 👍🏿